.github/workflows/md-links.yml

```diff
@@ -0,0 +1,17 @@
+# GitHub workflow file
+name: Check links in markdown
+
+on:
+  push
+
```

Comment:
I would add
    paths: '**.md'

Reply:
I actually did not want to do that, because I wanted to trigger the check, even if there are no changes to md file. The reason is that external links can break, even if we don't change the md file itself. This is to give an opportunity to catch it. 
Now, there is a good point as this check should run scheduled, daily - in case a link does break.